doc/source/BuildingRunningTesting/BuildRunLandDA.rst

```diff
@@ -0,0 +1,306 @@
+
+.. code-block:: console
+
+   _______________________________________________________________
+   Start Epilog on node h24c45 for job 56463665 :: Fri Mar  1 22:38:34 UTC 2024
+   Job 56463665 finished for user Gillian.Petro in partition hera with exit code 0:0
+   _______________________________________________________________
+   End Epilogue Fri Mar  1 22:38:34 UTC 2024
+
+The experiment has successfully completed when all tasks say SUCCEEDED under STATE. Other potential statuses are: QUEUED, SUBMITTING, RUNNING, and DEAD. Users may view the ``slurm-########.out`` files to determine why a task may have failed. 
+
+.. _run-batch-script:
+
+Run Without Rocoto
+--------------------
+
+Users may choose not to run the workflow with uwtools and Rocoto. To run the :term:`J-jobs` scripts in the ``jobs`` directory, navigate to the ``parm`` directory and edit ``run_without_rocoto.sh`` (e.g., using vim or preferred command line editor). Users will likely need to change the ``MACHINE``, ``ACCOUNT``, and ``EXP_BASEDIR`` variables to match their system. Then, run ``run_without_rocoto.sh``:
+
+.. code-block:: console
+
+   cd $LANDDAROOT/land-DA_workflow/parm
+   sbatch run_without_rocoto.sh
+
+Check Experiment Output
+=========================
+
+As the experiment progresses, it will generate a number of directories to hold intermediate and output files. The directory structure for those files and directories appears below:
+
+.. COMMENT: In the future: It will create an experiment directory in ``$LANDDAROOT/landda_expts/EXP_NAME`` to hold experiment output. (Note that ``$EXP_NAME`` was set in ``land_analysis.yaml``.)
+
+.. code-block:: console
+
+   $LANDDAROOT: Base directory
+    ├── land-DA_workflow(<CYCLEDIR>): Home directory of the land DA workflow
+    ├── com
+    │     ├── landda (<NET>)
+    │     │     └── vX.Y.Z (<model_ver>)
+    │     │     │     └── DA_<forcing> (<OUTDIR>)
+    │     │     │           ├── DA: Directory containing the output files of JEDI run
+    │     │     │           └── mem000: Directory containing the output files
+    │     └──  output
+    │           └──  logs
+    │                 └── run_<forcing> (<LOGDIR>): Directory containing the log file of the Rocoto workflow
+    └── workdir(<WORKDIR>)
+          └── run_<forcing>
+                └── mem000: Working directory
+
+Each variable in angle brackets is the name for the directory defined in the file ``land_analysis.yaml``. In the future, this directory structure will be further modified to meet the :nco:`NCO Implementation Standards<>`. 
+
+.. COMMENT: What is EXP_NAME used for? Why are the slurm log files ending up in parm?
```

Comment:
- `EXP_NAME` is not used to create the directory structure but used to check whether or not it is `openloop` in the job scripts.
- The log files are renamed to `<task>.log` and move to `com/output/logs`.